frontend/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -489,7 +479,4 @@ export default connect<Props, OwnProps, _, _, _, _>(
   mapStateToProps,
   mapDispatchToProps,
   null,
-  {
-    pure: false,
-  },
```

Comment:
I didn't notice any problems with the mapping UI during testing. The component is-rerendered when the mouse position changes, so it's probably hard to get an outdated mapping view.

Reply:
Hmm, I couldn't find out why this was added. It was added as part of the Custom Viewports PR, but I didn't see any specific reason. I agree that it should be fine :)